en/docs/observability/observability-overview.md

```diff
@@ -0,0 +1,72 @@
+# Observability Overview
+
+Observability in Choreo provides the capability to visualize and monitor the performance of services deployed on Choreo. Choreo has in-built support to view the overall status, latencies, throughput data,  diagnostic data, and logs. Hence, you can use Choreo Observability to detect and troubleshoot anomalies in your services efficiently. 
+
+## Dashboard
+Choreo observability dashboard gives you more than one way to monitor your services. This section will introduce the dashboards. 
+
+### Dashboard overview
+
+![Dashboard Overview](../assets/img/observability/overview-overall.png){: style="height:55%;width:95%"}
+
+The Observability dashboard allows you to:
+
+1. Observe the throughput and latencies of requests served over a given period.
+2. View the logs generated over a given period.
+3. Observe the flame graph (Diagnostics view) generated over a given period.
+4. View the low-code diagram.
+5.  Trace requests.
+
+!!! note
+    By default, the observability view is private and is only visible to the service owner. If you need to share it, toggle the **Share On** switch, copy the observability URL and share it with the required parties.
+    
+    ![Options Bar](../assets/img/observability/options-bar.png){: style="height:120%;width:90%"}
+
+### Throughput and latency graphs
+
+![Throughput and Latency Graph](../assets/img/observability/throughput-and-latency.png){: style="height:45%;width:70%"}
```

Comment:
Crop the image at the bottom as it contains unnecessary white space.

<img width="1273" alt="image" src="https://user-images.githubusercontent.com/16300038/120167320-38ee9f00-c21b-11eb-8590-fa7befdf58bb.png">

Reply:
done.